creator-node/test/nodesync.test.js

```diff
@@ -11,7 +11,11 @@ const models = require('../src/models')
 const { getApp, getServiceRegistryMock } = require('./lib/app')
 const { getLibsMock } = require('./lib/libsMock')
 const libsMock = getLibsMock()
-const { createStarterCNodeUser, testEthereumConstants, destroyUsers } = require('./lib/dataSeeds')
+const {
```

Comment:
sorry this diff is hard to follow - it's mainly adding 2 new tests and refactoring to dry code